Makefile

```diff
@@ -126,11 +136,13 @@ gen_key_cache: install_rs_build_toolchain
 		--features=$(TARGET_ARCH_FEATURE),shortint,internal-keycache -p tfhe
 
 .PHONY: build_core # Build core_crypto with and without experimental features
-build_core: install_rs_build_toolchain
+build_core: install_rs_build_toolchain install_rs_build_toolchain_nightly
```

Comment:
I think you could have used the nighlty check toolchain instead ? Or we can rename the check toolchain to nightly toolchain